### PR TITLE
docs: fixes 3084

### PR DESCRIPTION
**Pull request by melissahenderson**
This PR covers doc issues that were discussed during a meeting with Max on Nov. 05, 2024.

@mkurapov Tagging you because of the YAML updates. Also, the OpenAPI spec for IdP is missing `Continue Grant POST /interact/{id}/{nonce}`. Is that intentional or should I create an issue for it?

### CODE REVIEW

**Comment by mkurapov** [comment]
> Also, the OpenAPI spec for IdP is missing Continue Grant POST /interact/{id}/{nonce}. Is that intentional or should I create an issue for it?

Continue Grant POST is for the Open Payments Auth API, so shouldn't be in the IdP spec

File: packages/backend/src/openapi/specs/webhooks.yaml [hunk 1]
```diff
@@ -358,10 +358,10 @@ components:
         value:
           type: string
           format: uint64
-          description: 'The value is an unsigned 64-bit integer amount, represented as a string.'
+          description: 'An unsigned 64-bit integer amount, represented as a string.'
         assetCode:
           type: string
-          description: The assetCode is a code that indicates the underlying asset. This SHOULD be an ISO4217 currency code.
+          description: A code that indicates the underlying asset. This SHOULD be an ISO4217 currency code.
```
Comment (mkurapov) [inline packages/backend/src/openapi/specs/webhooks.yaml:364]: Maybe we can mention that it should be `an ISO4217` if possible. Kind of like we have in the docs page for assets:

"The asset code, generally an ISO 4217 currency code where available."

File: packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx [hunk 2]
```diff
@@ -194,11 +193,11 @@ Open Payments <LinkOut href="https://openpayments.dev/apis/wallet-address-server
 
 <div class="overflow-table">
 
-| Parameter | Required value | Description                                                                   |
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx:197]: was this auto-formatted?

Comment (melissahenderson) [reply packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx:197]: When we run prettier on the docs, it adds a bunch of extra spaces within all the markdown table columns and rows. That makes it harder for the writers to 'read' the tables and edit them later. Sarah's given us a temporary workaround where prettier will ignore the tables (actually, maybe all the doc files) and keep our original formatting. So this was me putting the table back to its original formatting.

File: packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx [hunk 3]
```diff
@@ -232,3 +231,84 @@ Additionally, the request must contain the `walletAddressId` of the wallet addre
...
+        alg
+        crv
+        kid
+        kty
+        x
+      }
+      createdAt
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (mkurapov) [reply packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx]: Still applicable ^

[the remark this answers is not in the document]
Comment (melissahenderson) [reply packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx]: Can you check the file again? I've updated the example response.

File: packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx [hunk 4]
```diff
@@ -232,3 +231,60 @@ Additionally, the request must contain the `walletAddressId` of the wallet addre
...
+      "success": true,
+      "walletAddressKey": {
+        "id": "f2953571-f10c-44eb-ab41-4450a7ad6771",
+        "walletAddressId": "695e7546-1803-4b45-96b6-6a53f4082018",
+        "revoked": true,
+        "createdAt": "2023-03-03T09:26:41.424Z"
+      }
+    }
+  }
+}
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx:287]:
```suggestion
{
  "data": {
    "revokeWalletAddressKey": {
      "walletAddressKey": {
        "id": "f2953571-f10c-44eb-ab41-4450a7ad6771",
        "revoked": true,
        "walletAddressId": "695e7546-1803-4b45-96b6-6a53f4082018",
        "createdAt": "2023-03-03T09:26:41.424Z"
      }
    }
  }
}
```

**Review by brad-dow — APPROVED** [review]

**Pull request merged**